test/parallel/test-timers-refresh-in-callback.js

```diff
@@ -0,0 +1,16 @@
+// Flags: --expose-internals
```

Comment:
Why is this needed?

Reply:
Thanks @mscdex — it's not, I just copy & pasted `test-timers-refresh` to create this.